src/qt/optionsdialog.cpp

```diff
@@ -185,6 +186,13 @@ void OptionsDialog::setModel(OptionsModel *_model)
     /* Display */
     connect(ui->lang, static_cast<void (QValueComboBox::*)()>(&QValueComboBox::valueChanged), [this]{ showRestartWarning(); });
     connect(ui->thirdPartyTxUrls, &QLineEdit::textChanged, [this]{ showRestartWarning(); });
+
+
+	/* VBK BFI */
+	//connect(ui->bfiDataSourceUrls, &QLineEdit::textChanged, this, &OptionsDialog::bfiChanged );
+	/* VBK BFI */
```

Comment:
is this PR a work-in-progress?

Reply:
No.
That is a reminder to me if we later add the capability to write conf files.

Reply:
Removed future proofing code from production release.